src/data/registry.xml

```diff
@@ -326,6 +327,12 @@
       <long_name>current timestep number</long_name>
       <initial_value>0</initial_value>
     </variable>
+    <!-- Date and time variables -->
+    <variable local_name="calday"
+              standard_name="fractional_calendar_days_on_end_of_current_timestep"
```

Comment:
Thanks for bringing this up, good question... This is used to store the result of `time_manager.F90::get_curr_calday()`, and the range is `(1.0, 366.0)`. So it is only reset once we reach the first day of a new year. I prefer `days` because of this but please let me know if there's a better alternate name.

Reply:
Thanks for the explanation!  I agree that it would be better described as `days` then, so feel free to leave this as-is!